moped-editor/src/views/projects/projectView/ProjectComponents/CreateComponentDrawTools.js

```diff
@@ -57,6 +57,7 @@ const ComponentDrawTools = ({
       type: "delete_drawn_features",
       payload: deletedFeaturesArray,
     });
+    setIsDrawing(false);
```

Comment:
This fixes a bug where lines or points were not appearing on the map immediately after deleting a drawn point or line. It was creating a state where users would have to interact with the draw tools again to see the layers to select from which is not what we want.